docs/bot-usage.md

```diff
@@ -13,7 +13,82 @@ For example, to check if the bot is alive you can write this comment:
 
 And the bot will reply to you.
 
-## Creating experiments
+## Table of Contents
+
+[h-toc]: #table-of-contents
+
+* [Tutorial: creating an experiment for a PR][h-tutorial]
+* [Available experiment modes][h-experiment-modes]
+* [Automatic experiment names][h-experiment-names]
+* Commands reference:
+  * [Creating experiments][h-cmd-run]
+  * [Editing experiments][h-cmd-edit]
+  * [Aborting experiments][h-cmd-abort]
+* Troubleshooting:
+  * [Regenerating a report if it failed][h-troubleshooting-retry-report]
+  * [Reload the list of GitHub team members][h-troubleshooting-reload-teams]
+
+## Tutorial: creating an experiment for a PR
+
+[h-tutorial]: #tutorial-creating-an-experiment-for-a-pr
+
+When you want to create an experiment to check if a PR is breaking some
+third-party code, the first thing is to request a **try build** to get a valid
+toolchain. You can do that with the `@bors try` GitHub comment.
+
+Once the try build is done, you can get the SHA of the start and end commits.
+The end commit is the commit by bors used for the try build, and the start
+commit is the first parent of that (another merge commit by bors). You must
```

Comment:
It may make sense to say that in the message posted by `bors try`

>  :hourglass: Trying commit XXXXXXX with merge YYYYYYY...

`end` is full hash for YYYYYYY, and `start` is full hash of the first ancestor of `XXXXXXX` that's not from your PR.

(I personally never figured out what the old description meant without looking at existing PRs with successful calls to craterbot as examples.)

Reply:
Determining start and end automatically will probably be more reliable than manually.
E.g. something like `craterbot --this-pr` could search for the last messages from `bors try` and extract the relevant commit hashes from them.

Reply:
> Determining start and end automatically will probably be more reliable than manually.
E.g. something like craterbot --this-pr could search for the last messages from bors try and extract the relevant commit hashes from them.

Yep, that's planned (#250).

Reply:
@petrochenkov reworded the section a bit, is it more clear now?